commit 6753629087b5452c3232e7ab1bd6409ac204c51f
Author: Puyan Lotfi <puyan@puyan.org>
Date:   Sun Feb 11 01:32:34 2024 -0500

    Add AArch64 deps to CMakeLists.txt

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 309855e0c..33b8807a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -182,6 +182,9 @@ if(TRITON_BUILD_PYTHON_MODULE)
     LLVMPasses
     LLVMX86CodeGen
     LLVMX86AsmParser
+    LLVMAArch64CodeGen
+    LLVMAArch64AsmParser
+
     LLVMNVPTXCodeGen
     # LLVMNVPTXAsmPrinter
     LLVMAMDGPUCodeGen
